hackathon_site/review/views.py

```diff
@@ -1,3 +1,99 @@
-from django.shortcuts import render
+from datetime import datetime, timedelta
 
-# Create your views here.
+from django.urls import reverse_lazy
+from django.shortcuts import redirect
+from django.views.generic.edit import FormView
+from django.contrib.auth.mixins import UserPassesTestMixin
+
+from django.template.loader import render_to_string
+from django.core import mail
+
+from review.forms import MailerForm
+from review.models import Review
+from hackathon_site import settings
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MailerView(UserPassesTestMixin, FormView):
+    template_name = "review/send-decisions.html"
+    success_url = reverse_lazy("admin:send-decision-emails")
+    form_class = MailerForm
+
+    def test_func(self):
+        return self.request.user.is_superuser
+
+    def form_valid(self, form, **kwargs):
+        """
+        On form submission valid get the data and send emails depending on the status.
+
+        Accepted / Waitlisted / Rejected emails will be sent for applications reviewed
+        (with last_updated_date) between date_start and date_end. The number of emails
+        for that status that will be sent is dictated by the quantity.
+        """
+        date_start = form.cleaned_data["date_start"]
```

Comment:
```python
date_start = datetime.combine(
    form.cleaned_data["date_start"], datetime.min.time()
).replace(tzinfo=settings.TZ_INFO)
date_end = datetime.combine(
    form.cleaned_data["date_end"], datetime.max.time()
).replace(tzinfo=settings.TZ_INFO)
```

Since the queryset expects a timezone-aware datetime object for `updated_at`, it's throwing warnings because it's getting a plain date object. This fixes that